File: sentimental-mario-less/mario.py
from cs50 import get_int

while True:
    h = get_int("Heyght: ")
    if h > 0 and h < 9:
        break

for i in range(h):
    for j in range(h):

        # print а "space"
        if j < (h - 1 - i):
            print(" ", end="")

        # print а "brick"
        else:
            print("#", end="")
    print()

File: mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int h;
    do
    {
        // pyramid height query
        h = get_int("Height: ");
    }
    while (h < 1 || h > 8);

    // for each row
    for (int i = 0; i < h; i++)
    {

        // for each column
        for (int j = 0; j < h; j++)
        {
            //print а "space"
            if (j < h - 1 - i)
            {
                printf(" ");
            }
            //print а "brick"
            else
            {
                printf("#");
            }
        }
        //move to next row
        printf("\n");
    }
}
